Jellyfin.Server/Migrations/MigrationRunner.cs

```diff
@@ -75,6 +75,10 @@ public static void RunPreStartup(ServerApplicationPaths appPaths, ILoggerFactory
 
             var xmlSerializer = new MyXmlSerializer();
             var migrationConfigPath = Path.Join(appPaths.ConfigurationDirectoryPath, MigrationsListStore.StoreKey.ToLowerInvariant() + ".xml");
+            if (!File.Exists(migrationConfigPath))
+            {
+                return;
+            }
             var migrationOptions = (MigrationOptions)xmlSerializer.DeserializeFromFile(typeof(MigrationOptions), migrationConfigPath)!;
```

Comment:
It still needs to run
```suggestion
            var migrationOptions = File.Exists(migrationConfigPath)
                 ? (MigrationOptions)xmlSerializer.DeserializeFromFile(typeof(MigrationOptions), migrationConfigPath)!
                 : new MigrationOptions();

```